cmd/gokeyless/gokeyless.go

```diff
@@ -93,41 +212,69 @@ func main() {
 			log.Info("already configured; exiting")
 		}
 		os.Exit(0)
+	case outputConfigMode:
+		b, err := yaml.Marshal(config)
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Print(string(b))
+		os.Exit(0)
 	}
 
 	// If we make it here we need to ask for user input, so we need to give up
 	// and log an error instead (in case the server is running as a daemon).
 	// Failing hard with an error message makes the problem obvious, whereas a
 	// daemon blocked waiting on input can be hard to debug.
 	if needNewCertAndKey() {
-		log.Error("the server cert/key need to be generated; run the server with either the -config-only or -manual-activation flag to generate the pair")
-		os.Exit(1)
+		if needInteractivePrompt() {
+			log.Error("the server cert/key need to be generated; set the hostname, zone_id, and origin_ca_key values in your config file, or run the server with either the -config-only or -manual-activation flag to generate the pair interactively")
```

Comment:
log.Fatal?

Reply:
The first lines of initializeServerCertAndKey are:
```
if needsInteractivePrompt() {
  interactivePrompt()
}
```
but we've cut it off here. Is initializeServerCertAndKey called somewhere else?

Reply:
Yes, it's called when the `--config-only` flag is provided, which is handled earlier in the flow.

The idea is to only run ever run the interactive flow if that flag is provided, while also still attempting to auto bootstrap if we have all the input we need from the user via config.

Can switch to `log.Fatal` if it's preferred - the existing conventions are mixed.